docs/sphinx/using/python.rst

```diff
@@ -33,12 +37,20 @@ If all goes well, the state measured with the highest probability should be our
 Variational Quantum Eigensolver
 --------------------------------
 
-Let's take a look at how we can use CUDA Quantum's built-in `vqe` module to run our own custom VQE routines! Given a parameterized quantum kernel, a system spin Hamiltonian, and one of CUDA Quantum's optimizers, `cudaq.vqe` will find and return the optimal set of parameters that minimize the energy, <Z>, of the system.
+The Variational Quantum Eigensolver (VQE) algorithm, originally proposed in
+`this publication <https://arxiv.org/abs/1304.3061>`__, 
+is a hybrid algorithm that can make use of both quantum and classical resources.
+
+Let's take a look at how we can use CUDA Quantum's built-in `vqe` module to run our own custom VQE routines! 
+Given a parameterized quantum kernel, a system spin Hamiltonian, and one of CUDA Quantum's optimizers, 
+`cudaq.vqe` will find and return the optimal set of parameters that minimize the energy, <Z>, of the system.
+
+The code block below represents the contents of the file `simple_vqe.py` from the `CUDA Quantum Github repository<https://github.com/NVIDIA/cuda-quantum/blob/main/docs/sphinx/examples/python/simple_vqe.py>`__. 
```

Comment:
This link will be inaccurate if someone is browsing another version of the docs than latest. 
You could either remove this sentence, and we can look into setting up a button along the lines of "Show source" for each code block in the future when we actually have a source file for each code snippet, or you should give a relative link to ensure you link the version of the file that is indeed show here. For the latter option, I think you should be able to do something like :doc:`../examples/python/simple_vqe.py`, or ... :doc:`this file <../examples/python/simple_vqe.py>`.